datacamp_python_project_spec_lefthandedness.ipynb

```diff
@@ -0,0 +1,255 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7ffbb86f7c50>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# create a new column for birth year of each age\n",
+    "lefthanded_data['Year'] = 1986 - lefthanded_data['Age'] # the study was done in 1986\n",
+    "\n",
+    "# create a new column for the average of male and female\n",
+    "lefthanded_data['Mean_lh'] = lefthanded_data[['Female', 'Male']].mean(axis = 1)\n",
+    "\n",
+    "\n",
+    "plt.figure()\n",
+    "ax = plt.gca()\n",
```

Comment:
I like using `fig, ax = plt.subplots()` instead of these two lines, since it creates the figure and axes objects in the same function call, and can be used to create and assign many subplots at once all up front. I checked the datacamp visualization lectures and they actually only teach the `plt.` matlab-like approach of plotting into the current axes instead of the object orient matplotlib approach =(

Reply:
Sounds good, this is more general.